yash.md

```diff
@@ -2,7 +2,85 @@
 
 _A combination index, glossary, and high level guided tour for the Stanford Digital Repository_
 
-* Access Rights for digital content
+Intended Audience: infrastructure team developers
+
+## Argo
+_The administrative discovery interface for Stanford's Digital Object Registry_
+- ask Andrew, John, JCoyne
+  * Specially chosen staff users (many of them accessioneers)
+
+## infrastructure-integration-test
+_A collection of Capybara integration tests for the SDR_
+- ask any infrastructure dev
+
+## Types of Digital objects
+_We started with more, but we're kinda down to three_
+- ask Andrew
+  * APO
+  * Collection
+  * Item
+
+## cocina-models
+_SDR data model written as syntactically validatable with dry-struct, dry-types and openapi_
+- ask JCoyne or JLitt
+
+## dor-services-app
+_In an effort to isolate our code that directly depends on Fedora3, the DSA was created.  It has grown to be a sort of kitchen sink nexus of a great deal of cross-application / cross-service needs, exposed as API calls._
+
+## Workflows
+
+### Workflow Server
+_a Rails App to track the workflows associated with an object, and the state of each step and an error string for failed steps_
+- ask JCoyne
+
+## Robots
```

Comment:
can we put this in https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md#robots ?